ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/Config.scala

```diff
@@ -522,6 +524,15 @@ object Config {
             "Experimental buffer for Ledger API streaming queries. Should not be used in production."
           )
           .action((_, config) => config.copy(enableInMemoryFanOutForLedgerApi = true))
+
+        // TODO ha: remove after stable
+        opt[Unit]("experimental-index-ha")
```

Comment:
```suggestion
        opt[Unit]("index-ha-unsafe")
```
In accordance with the hitherto used naming convention

Reply:
@meiersi-da @mziolekda 
I am good with both approaches. Can you agree on one? Then let me know, and I will change accordingly.

Reply:
@mziolekda is the owner of the code, so we go with his.

Reply:
renamed